### DISCUSSION TITLE
Multi Series line chart y axis values

### DISCUSSION THREAD

**Discussion by CJohnson25** [reaviz/reaviz#57] [Q&A]
I cannot seem to figure out why the y axis of my multi series line chart seems odd. The upper limit of the y axis seems to be the same as the highest value in the chart. I would suspect that it should provide some padding, to attempt to center the data vertically in the available space, which in turn I would suspect result in a y axis upper limit of something higher than the data set. 

![Screenshot 2021-10-29 225713](https://user-images.githubusercontent.com/6362561/139517952-92cd4e29-c497-4071-a42b-eb9aa8e20b2c.png)

**Comment by amcdnl** [comment]
Can you post a codesandbox example?
